src/SearchParameters/index.js

```diff
@@ -960,6 +1009,60 @@ SearchParameters.prototype = {
     return this.setQueryParameters(modification);
   },
   /**
+   * Remove a facet from the facets attribute of the helper configuration, if it
+   * is present.
+   * @method
+   * @param {string} facet facet name to remove
+   * @return {SearchParameters}
+   */
+  removeFacet: function removeFacet(facet) {
+    if (!this.isConjunctiveFacet(facet)) {
+      return this;
+    }
+
+    const newFacets = this.facets.slice();
+    newFacets.splice(newFacets.indexOf(facet), 1);
+    return this.setQueryParameters({
+      facets: newFacets
+    });
+  },
+  /**
+   * Remove a disjunctive facet from the disjunctiveFacets attribute of the
+   * helper configuration, if it is present.
+   * @method
+   * @param {string} facet disjunctive facet name to remove
+   * @return {SearchParameters}
+   */
+  removeDisjunctiveFacet: function removeDisjunctiveFacet(facet) {
+    if (!this.isDisjunctiveFacet(facet)) {
+      return this;
+    }
+
+    const newFacets = this.disjunctiveFacets.slice();
+    newFacets.splice(newFacets.indexOf(facet), 1);
+    return this.setQueryParameters({
```

Comment:
I would prefer as a user to not have to clear all refinements before remove a facet but that may be a bad shortcut?